.github/workflows/build_hub.yml

```diff
@@ -80,7 +85,11 @@ jobs:
 
       - name: Publish Integration Test Results
         uses: EnricoMi/publish-unit-test-result-action@v1
-        if: always()
+        # Per https://github.com/EnricoMi/publish-unit-test-result-action#support-fork-repositories-and-dependabot-branches
+        if: >
+          always() &&
```

Comment:
remove the `always()` since it's used in `&&` and it always returns `true` anyway
see https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#always

Reply:
I agree the always is needed, but I do want to keep the code as references in the instructions. The always does not hurt.